fake_switches/juniper/juniper_netconf_datastore.py

```diff
@@ -210,10 +217,11 @@ def parse_interfaces(self, conf, etree_conf):
             operation = resolve_operation(interface_node)
             if operation == "delete" and isinstance(port, AggregatedPort):
                 conf.remove_port(port)
+            if operation == "delete" and not isinstance(port, AggregatedPort) and not isinstance(port, VlanPort):
```

Comment:
Can we make sure instead that a Vlanport passing in this codepath raises a NotImplementedError rather than passing a comment ? A test would be writable for that ?